### ISSUE TITLE
1.1.0-beta not able to open model saved by 1.1.0-alpha1

### ISSUE THREAD

**Issue by pematt** [pgmodeler/pgmodeler#1830]
**Bug description**
When using 1.1.0-beta to open a model saved with 1.1.0-alpha1 I get the following error message:

"Could not load the database model file /home/user/code/dbs/example.dbm. Check the error stack to see details. You can try to fix it in order to make it loadable again."

How can I locate the exact place in the dbm file where this error is triggered? If it can find it I can provide an example dbm.

**Info about your desktop**
 - OS: Linux Mint
 - Version: 21.1
 - Window manager: default
 - pgModeler version: 1.1.0-beta
 - Qt version: 6.2.4

**Stacktrace / Debug info**

```
[0] src/mainwindow.cpp (2045)
  void MainWindow::showFixMessage(Exception&, const QString&)
    [ModelFileNotLoaded] Could not load the database model file `/home/mattias/code/lootquick/marketplacedb/marketplacedb.dbm'. Check the error stack to see details. Try to run `pgmodeler-cli --fix-model' in order to correct the structure of the file if that is the case.

[1] src/mainwindow.cpp (1259)
  void MainWindow::addModel(const QString&)
    [InvModelFileNotLoaded] Could not load file `/home/mattias/code/lootquick/marketplacedb/marketplacedb.dbm'. The same appears to be inconsistent or one of its dependencies (DTD files) has errors or is missing!

[2] src/mainwindow.cpp (1234)
  void MainWindow::addModel(const QString&)
    [InvModelFileNotLoaded] Could not load file `/home/mattias/code/lootquick/marketplacedb/marketplacedb.dbm'. The same appears to be inconsistent or one of its dependencies (DTD files) has errors or is missing!

[3] src/widgets/modelwidget.cpp (1763)
  void ModelWidget::loadModel(const QString&)
    [InvModelFileNotLoaded] Could not load file `/home/mattias/code/lootquick/marketplacedb/marketplacedb.dbm'. The same appears to be inconsistent or one of its dependencies (DTD files) has errors or is missing!

[4] src/databasemodel.cpp (3542)
  void DatabaseModel::loadModel(const QString&)
    [InvModelFileNotLoaded] Could not load file `/home/mattias/code/lootquick/marketplacedb/marketplacedb.dbm'. The same appears to be inconsistent or one of its dependencies (DTD files) has errors or is missing!

[5] src/xmlparser.cpp (84)
  void XmlParser::loadXMLFile(const QString&)
    [LibXMLError] Error while interpreting XML buffer at line 0 column 0.
Message generated by the parser: Element view content does not follow the DTD, expecting (schema , role? , tablespace? , comment? , tag? , appended-sql? , prepended-sql? , position , reference* , simplecol* , definition), got (schema role position reference) . 
       ** /home/mattias/code/lootquick/marketplacedb/marketplacedb.dbm

[6] src/xmlparser.cpp (115)
  void XmlParser::loadXMLBuffer(const QString&)
    [LibXMLError] Error while interpreting XML buffer at line 0 column 0.
Message generated by the parser: Element view content does not follow the DTD, expecting (schema , role? , tablespace? , comment? , tag? , appended-sql? , prepended-sql? , position , reference* , simplecol* , definition), got (schema role position reference) . 

[7] src/xmlparser.cpp (189)
  void XmlParser::readBuffer()
    [LibXMLError] Error while interpreting XML buffer at line 0 column 0.
Message generated by the parser: Element view content does not follow the DTD, expecting (schema , role? , tablespace? , comment? , tag? , appended-sql? , prepended-sql? , position , reference* , simplecol* , definition), got (schema role position reference) . 
```

**Comment by pematt** [comment]
When I allowed pgModeler to fix the model then it dropped all views and some relationships. Based on that then one could speculate that he error above may be related to views and relationships.

**Comment by rkhaotix** [comment]
Could you please send me a copy of your model? Views have changed considerably from 1.1.0-alpha1 to 1.1.0-beta. The ```pgmodeler-cli --fix-model``` operation can fix the majority of cases but it's not perfect. Maybe your model serves me as a reference to implement some patches on the fix process.

**Comment by pematt** [comment]
I was going to send you a part of my model, then [#1831](https://github.com/pgmodeler/pgmodeler/issues/1831) happened.

**Comment by pematt** [comment]
This is partly fixed in [884ab63](https://github.com/pgmodeler/pgmodeler/commit/884ab637103cade39ba59aa7065d7f4842f7ac78) since it can now import the views. It still says it can not import the model so it has to be fixed before opening it.
